### ISSUE TITLE
Global recourse packs

### ISSUE THREAD

**Issue by Freddie-06** [MultiMC/MultiMC5#3861]
### Role

I play with many instances, all with the same recourse pack

### Suggestion

I would like to have a folder, that automatically copies a file into every (new) instance, that is created.

### Benefit

Too make the time from clicking "new instance" to having a fully playable and personalized instance

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

**Comment by bbb651** [comment]
Adding to this, instead of copying the file over it can use symlinks which will reduce redundant file size and disk usage for the copying operations.

**Comment by spannerman79** [comment]
> use symlinks

MultiMC is a _portable_ application.

Furthermore on windows to create a symlink requires the use of elevated permissions on an OS where symlinking is known to be ...unstable at best.

No.

**Comment by Freddie-06** [comment]
> Furthermore on windows to create a symlink requires the use of elevated permissions
Oof. That would not be good for a Minecraft launcher, and may take away potential users

**Comment by bbb651** [comment]
You are right, I didn't take portability and other operating systems into account. I'm not sure how copying symlinks works but it would be nice if the end user could put a symlink instead resource pack and get the behavior I described, without drawbacks for other users.

**Comment by glowiak** [comment]
`ln -s`

**Comment by LinkZeldasGuard** [comment]
What about instead of using symlinks or copying the files over, could you not just redirect Minecraft to use a "global resoucepacks" folder outside of the "instances" folder? I mean like similar to the official Launcher where you can select your on repository for the installation. Wouldn't it be possible to do just that, but split it up into the seperate folders like "resourcepacks", "saves", etc, so you can assign one resourcepacks folder to all of your instances, that is, if you want of couse, I'd keep the current system and just make it optional.
